.travis.yml

```diff
@@ -1,18 +1,40 @@
-dist: xenial
-language: python
-python:
-  - 3.6
+language: c
+env:
+    matrix:
+        - DOCKER_IMAGE=nmstate/centos7-nmstate-dev TEST_TYPE=integ
+        - DOCKER_IMAGE=nmstate/fedora-nmstate-dev TEST_TYPE=integ
+        - DOCKER_IMAGE=nmstate/fedora-nmstate-dev TEST_TYPE=unit_code_style
+        - DOCKER_IMAGE=nmstate/fedora-nmstate-dev TEST_TYPE=unit_py27
+        - DOCKER_IMAGE=nmstate/fedora-nmstate-dev TEST_TYPE=unit_py36
+        - DOCKER_IMAGE=nmstate/fedora-nmstate-dev TEST_TYPE=unit_py37
 
 addons:
   apt:
     packages:
-    - libgirepository1.0-dev
+        - realpath
```

Comment:
It would be great to sort the list of packages by name.

Reply:
Done.